src/plugins/v-focus/directives/focus-item.ts

```diff
@@ -1,4 +1,4 @@
-import { DirectiveBinding } from '@vue/runtime-core';
+import { DirectiveBinding } from 'vue';
```

Comment:
question/suggestion: I'm confuse why this is `vue`... Is this because of the `webpack` alias? I would prefer `@vue/runtime-core` to be honest. Previously this was used, because we had to include the template parser in the build, which by default was not the case. I guess this would better be handled by importing the correct package.

Reply:
Reason is `vue/prefer-import-from-vue` rule